src/theme/components/button.ts

```diff
@@ -106,6 +156,9 @@ const callout = {
   _active: {
     bg: "brand.secondary",
   },
+  _dark: {
+    bg: "dark.brand.primary",
```

Comment:
I feel like I don't see this callout button changing in storybook on dark mode, but maybe just me?

Reply:
Correct.  There is no difference.  We left the callout button the same hot color in both `light` and `dark` modes.